router/xgress_edge_tunnel_v2/dialer.go

```diff
@@ -82,3 +82,10 @@ func (self *tunneler) Dial(params xgress.DialParams) (xt.PeerData, error) {
 
 	return peerData, nil
 }
+
+func (self *tunneler) Inspect(key string, timeout time.Duration) any {
+	if key == "ert-terminators" {
```

Comment:
I've seen this value a few times. Is there a reason it isn't a constant? It looks meaningful.

Reply:
fixed